cpp/src/arrow/compute/kernels/vector_sort.cc

```diff
@@ -1826,7 +1823,7 @@ const FunctionDoc top_k_doc(
      "at the end of the array.\n"
      "For floating-point types, NaNs are considered greater than any\n"
      "other non-null value, but smaller than null values."),
-    {"input"}, "TopKOptions");
+    {"input"}, "SelectKOptions::TopKDefault");
```

Comment:
Note the param here should be just the name of the options class, i.e. "SelectKOptions". Python uses this for introspection.

Reply:
Though, why do we still even have top_k_doc and bottom_k_doc?

Reply:
yeap it's old code